input/resources/au-core-medicationrequest.xml

```diff
@@ -248,8 +248,8 @@
         <key value="au-core-medreq-03"/>
         <severity value="error"/>
         <human value="Date shall be at least to day"/>
```

Comment:
change the human expression, see US Core Patient for exemplar

Reply:
Done